server/base/src/main/java/org/apache/accumulo/server/master/balancer/ChaoticLoadBalancer.java

```diff
@@ -162,7 +163,16 @@ public long balance(SortedMap<TServerInstance,TabletServerStatus> current, Set<K
     return 100;
   }
 
+  /**
+   * Initialize the TabletBalancer. This gives the balancer the opportunity to read the configuration.
```

Comment:
Don't need to duplicate the javadoc here. Most IDEs will inherit it from TabletBalancer